documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -156,73 +144,41 @@ claimRef:
   uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea
 ----
 
-. Deploy the Cluster Operator.
+. Deploy the Cluster Operator:
 +
-[source,shell,subs="+quotes,attributes"]
+[source,shell]
 ----
-kubectl create -f install/cluster-operator -n _my-project_
+kubectl create -f install/cluster-operator -n my-project
 ----
 
-. Recreate your cluster.
+. Recreate all `KafkaTopic` resources:
 +
-Follow the steps depending on whether or not you have all the `KafkaTopic` resources needed to recreate your cluster.
-+
---
-*_Option 1_*: If you have *all* the `KafkaTopic` resources that existed before you lost your cluster, including internal topics such as committed offsets from `__consumer_offsets`:
-
-. Recreate all `KafkaTopic` resources.
+[source,shell]
+kubectl apply -f <topic_configuration_file>
 +
-It is essential that you recreate the resources before deploying the cluster, or the Topic Operator will delete the topics.
+It is essential that you recreate the resources before deploying the cluster or the Topic Operator will delete the topics.
```

Comment:
This could be rephrased to something like: "It is important that you recreate the resources with the same configurations, or the Topic Operator will try to update them in Kafka."